crates/orderbook/src/api.rs

```diff
@@ -41,6 +41,7 @@ mod version;
 
 pub fn handle_all_routes(
     database: Postgres,
+    database_replica: Postgres,
```

Comment:
That we are using a replica is an implementation detail. The important part is that it only allows reading.
```suggestion
    db_write: Postgres,
    db_read: Postgres,
```